thirdparty/rabbitmq-server/zimbra-rabbitmq-server/rpm/SPECS/rabbitmq-server.spec

```diff
@@ -0,0 +1,61 @@
+Summary:            Zimbra's RabbitMQ server
+Name:               zimbra-rabbitmq-server
+Version:            VERSION
+Release:            1zimbra8.8b1ZAPPEND
+License:            MPLv2.0 and MIT and ASL 2.0 and BSD
+Source:             %{name}-%{version}.tar.xz
+Source1:            rabbitmq-env.conf
+Patch0:             rabbitmq-script-wrapper.patch
+Packager:           Zimbra Packaging Services <packaging-devel@zimbra.com>
+AutoReqProv:        no
+URL:                https://github.com/rabbitmq/rabbitmq-server
+BuildRequires:      zimbra-erlang >= 25.0.3-1zimbra8.8b1ZAPPEND
+BuildRequires:      zimbra-openssl-devel >= 1.1.1q-1zimbra8.7b4ZAPPEND, zimbra-openssl-libs >= 1.1.1q-1zimbra8.7b4ZAPPEND
+BuildRequires:      gzip, sed, zip, rsync
+Requires:           zimbra-erlang >= 25.0.3-1zimbra8.8b1ZAPPEND
```

Comment:
does this mean it's runtime dependency and line no 12 is build time dependency?

Reply:
yes it is runtime dependency